Ryujinx.HLE/HOS/Services/Am/AppletOE/ApplicationProxyService/ApplicationProxy/IApplicationFunctions.cs

```diff
@@ -330,14 +348,49 @@ public ResultCode QueryApplicationPlayStatisticsByUid(ServiceCtx context)
             return (ResultCode)QueryPlayStatisticsManager.GetPlayStatistics(context, true);
         }
 
+        [Command(120)] // 5.0.0+
+        // ExecuteProgram(ProgramSpecifyKind kind, u64 value)
+        public ResultCode ExecuteProgram(ServiceCtx context)
+        {
+            ProgramSpecifyKind kind = (ProgramSpecifyKind)context.RequestData.ReadUInt32();
+
+            // padding
+            context.RequestData.ReadUInt32();
+
+            ulong value = context.RequestData.ReadUInt64();
```

Comment:
```suggestion
            ulong programIndex = context.RequestData.ReadUInt64();
```
And same in all other functions who use this value

Reply:
This was marked as resolved - is it because this value can have different meaning depending on `kind`?